metricbeat/autodiscover/builder/hints/metrics.go

```diff
@@ -198,6 +208,14 @@ func (m *metricHints) getMetricPath(hints common.MapStr) string {
 	return builder.GetHintString(hints, m.Key, metricspath)
 }
 
+func (m *metricHints) getUserName(hints common.MapStr) string {
```

Comment:
nit pick only: isn't it correct in English to use a single word "username" instead two ones "user Name"?